spec/models/sitemap_builder_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
```

Comment:
❤️ these tests - really clear 😄 

Is it testing whatever is up with the `xmldump` business?

Reply:
It tests the output of the sitemap builder methods based on a stubbed version of the data we expect to get back, but not the call itself. The call takes a wee while so testing the actual return value of the xmldump so it would make the tests take waaayyy longer and it would need to be a test on the Sign model anyway.